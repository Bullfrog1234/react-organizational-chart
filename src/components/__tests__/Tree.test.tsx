import Tree from "../Tree";
import TreeNode from "../TreeNode";
import React from "react";
import renderer from "react-test-renderer";

describe("Tree", () => {
  it("is exported", () => {
    expect(Tree).toBeTruthy();
  });

  it("Renders", () => {
    const component = renderer.create(
      <Tree label={<div>Root</div>}>
        <TreeNode label={<div>Child</div>}>
          <TreeNode label={<div>Grand Child</div>} />
        </TreeNode>
      </Tree>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
