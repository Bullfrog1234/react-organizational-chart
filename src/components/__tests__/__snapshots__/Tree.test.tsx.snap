// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tree Renders 1`] = `
<ul
  className="css-13i5t6k"
>
  <li
    className="css-1bmag6y"
  >
    <div>
      Root
    </div>
    <ul
      className="css-s8xjl5"
    >
      <li
        className="css-1bmag6y"
      >
        <div>
          Child
        </div>
        <ul
          className="css-s8xjl5"
        >
          <li
            className="css-1bmag6y"
          >
            <div>
              Grand Child
            </div>
          </li>
        </ul>
      </li>
    </ul>
  </li>
</ul>
`;
